### PR TITLE
Update package

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe, LGTY?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +20,16 @@
     "prepublish": "cake build"
   },
   "dependencies": {
-    "leadconduit-fields": ">=2.6.0",
     "underscore": "^1.7.0",
     "underscore.string": "^3.0.2",
-    "leadconduit-integration": "0.0.1"
+    "leadconduit-integration": "0.0.1",
+    "mime-content": ">=0.0.6",
+    "lodash": "^3.10.0"
```
Comment (alexkwolfe) [inline package.json]: Why did you add lodash?

File: package.json [hunk 2]
```diff
@@ -20,17 +20,16 @@
...
   },
   "devDependencies": {
     "coffee-script": ">= 1.8.0",
     "mocha": ">= 1.21.0",
     "chai": "^1.10.0",
-    "leadconduit-types": ">=4.2.0",
-    "lodash": "^3.10.0",
-    "mime-content": ">=0.0.6"
+    "leadconduit-fields": ">=2.6.0"
   }
```
Comment (alexkwolfe) [inline package.json]: The lodash dependency is [used only in the specs](https://github.com/activeprospect/leadconduit-integration-lendio/search?utf8=✓&q=lodash). Which means it should stay in devDependencies. 

But since this project already uses the underscore module, which is essentially the same thing, I recommend you remove lodash entirely and rely solely on underscore.

Comment (ros3bud) [inline package.json]: Shoot. Yeah, I see that. I thought it was used in the src.

Comment (ros3bud) [inline package.json]: I need lodash for merging in the outbound spec.

**Comment by alexkwolfe** [comment]
LGTM
